snapshot/manifest.go

```diff
@@ -28,6 +28,8 @@ type Manifest struct {
 	RootEntry *DirEntry `json:"rootEntry"`
 
 	RetentionReasons []string `json:"-"`
+
+	Tags map[string]string `json:"tags,omitempty"`
```

Comment:
It may be a good idea to convert the snapshot tags to "Manifest labels", that way the same mechanisms used to query and filter manifests can be used to query/list/filter snapshot manifest. See `snapshot.SaveSnapshot()`.

Then, there's a question of whether or not it is necessary to explicitly store the tags here in the snapshot manifest.

Reply:
Let's ask upstream for feedback on this one, there may be an alternate easy way of doing it.